src/site/content/en/blog/module-workers/index.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "Threading the Web with Module Workers"
+subhead: |
+  Moving heavy lifting into background threads is now easier with JS Modules in Workers.
```

Comment:
Is 'JS Modules' the official name of the spec or is it actually 'JavaScript Modules'. Use the exact name and spelling

Also if you're not using the spec name it should be . 'JavaScript modules in workers.' (Notice the capitalization.)

Reply:
"JavaScript Modules" is the official name